src/SSW.Rewards.Application/Quizzes/Common/QuizDetailsDto.cs

```diff
@@ -33,6 +33,7 @@ public class QuestionAnswerDto : IMapFrom<QuizAnswer>
 {
     public int QuestionAnswerId { get; set; }
     public string Text { get; set; } = string.Empty;
+    public bool IsCorrect { get; set; }
```

Comment:
Exclude from mapping or we are giving away our secrets!

Reply:
Done